File: coding/16th May 2025/Height Checker.java
//Height Checker
class Solution 
{
    public int heightChecker(int[] heights) 
    {
        int[] original = new int[heights.length];
        for(int i = 0; i < heights.length; i++)
            original[i] = heights[i];
        Arrays.sort(heights);
        int res = 0;
        for(int i = 0; i < heights.length; i++)
            if(original[i] != heights[i])
                res++;
        return res;
    }
}

File: Coding in Python/07th February 2026/Height Checker.py
#Height Checker
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        result = 0
        temp = []
        for ele in heights:
            temp.append(ele)
        temp.sort()
        for i in range(len(temp)):
            if heights[i] != temp[i]:
                result += 1
        return result
